core/src/themes/ionic/ionic.theme.default.scss

```diff
@@ -10,85 +10,211 @@
 // Default Ionic Colors
 // -------------------------------------------------------------------------------------------
 // Color map should provide
-// - base: main color to be used.
-// - contrast: Color that will provide readable text against the base color
-// - shade: 12% darker version of the base color (mix with black)
-// - tint: 10% lighter version of the base color (mix with white)
+// - bold: a map of the bold color variations
+// - subtle: a map of the subtle color variations
+//
+// Each hue color map should provide
+// TODO: rename base to background?
```

Comment:
I think renaming it to background would make a lot more sense with how the design tokens are used but this would be a breaking change if we also updated the css variables: https://ionicframework.com/docs/theming/colors#new-color-creator